koku/masu/database/ocp_report_db_accessor.py

```diff
@@ -1178,6 +1178,26 @@ def get_openshift_topology_for_multiple_providers(self, provider_uuids):
 
         return topology_list
 
+    def get_filtered_openshift_topology_for_multiple_providers(self, provider_uuids, start_date, end_date):
+        """Return a dictionary with 1 or more Clusters topology."""
+        topology_list = []
+        for provider_uuid in provider_uuids:
+            cluster = self.get_cluster_for_provider(provider_uuid)
+            nodes_tuple = self.get_nodes_trino(provider_uuid, start_date, end_date)
+            pvc_tuple = self.get_pvcs_trino(provider_uuid, start_date, end_date)
+            topology_list.append(
+                {
+                    "cluster_id": cluster.cluster_id,
+                    "cluster_alias": cluster.cluster_alias,
+                    "provider_uuid": provider_uuid,
+                    "nodes": [node[0] for node in nodes_tuple],
+                    "resource_ids": [node[1] for node in nodes_tuple],
+                    "persistent_volumes": [pvc[0] for pvc in pvc_tuple],
```

Comment:
Why are we excluding pvcs and projects? These are in the `get_openshift_topology_for_multiple_providers`:
```
                    "persistent_volume_claims": [pvc[1] for pvc in pvc_tuple],
                    "projects": [project for project in project_tuple],
```

Reply:
We don't actually use the PVC names or projects for any OCP on Cloud matching, and the original method was over-engineered in that sense.